<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>click button</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="discriptionInput">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>711</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Image Set</string>
   </property>
   <widget class="QLabel" name="imageFrame">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>141</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(220, 220, 220);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="browserImageButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>160</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browser Image</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="widthInput">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="heightInput">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="xInput">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="yInput">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>width</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>height</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLabel" name="titleCheck">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 2, 6);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>441</width>
     <height>171</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="PressKey">
    <attribute name="title">
     <string>Press Key</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>371</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Required</string>
     </property>
     <widget class="QComboBox" name="modifiers">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>L CTRL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>R CTRL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>L SHIFT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>R SHIFT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>L ALT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>R ALT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ENTER</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="keyInput">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Key</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="keysCheck">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Mouse click</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>371</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Required</string>
     </property>
     <widget class="QComboBox" name="mouseCombo">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>left</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>right</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>click button</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="muoseCheck">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>540</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>540</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>450</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Priority</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="priority">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>450</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
